src/widgets/forms.js

```diff
@@ -1295,10 +1295,11 @@ export function makeSelectForOptions (
         predicate +
         '.'
     )
-  }
+  }/* See https://github.com/solid/solid-ui/issues/453
```

Comment:
**suggestion** remove it, instead of commenting it out. Reference the issue in the PR and the commit message to document why

Reply:
Agree